apps/ngs_roi/project_interval.cpp

```diff
@@ -237,7 +237,7 @@ void IntersectBed::finishContig()
     if (options.verbosity >= 2)
         std::cerr << "finishContig()\n";
 
-    while (!empty(bedRecords))
+    while (!bedRecords.empty())
     {
```

Comment:
Why this change? Is the global function broken? If yes, we should fix the global function...